netci.groovy

```diff
@@ -219,6 +219,8 @@ commitPullList.each { isPr ->
       def myJob = job(jobName) {
         description("Windows ${configuration} tests on ${buildTarget}")
         steps {
+          batchfile('tasklist /FI "IMAGENAME eq devenv.exe"')
+          batchfile('taskkill /FI "IMAGENAME eq devenv.exe"')
```

Comment:
Shouldn't we add this as an after step? 

If the theory is VS integration is leaving devenv processes around then not killing after is polluting other jobs that may share our machine pool + rely on VS.